list/letters-filter/style/letters-filter.scss

```diff
@@ -0,0 +1,21 @@
+
+
+[data-focus='letters-filter-bar'] {
+ /* float: left;
+  margin-right: 30px;*/
+  [data-focus='letter-element'] {
+    display: inline-block;
+    padding: 0 10px 0 10px;
+    background-color: #fff;
+    color: #3CC098;
+  }
+  [data-focus='letter-element'].disabled {
+    color: #999;
+    background-color: #ccc;
```

Comment:
Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.